util/util.go

```diff
@@ -643,6 +643,33 @@ func CreateDiskPathReplicaSubdirectory(path string) error {
 	return nil
 }
 
+func DeleteDiskPathReplicaSubdirectoryAndDiskCfgFile(path string) error {
+	nsPath := iscsi_util.GetHostNamespacePath(HostProcPath)
+	nsExec, err := iscsi_util.NewNamespaceExecutor(nsPath)
+	if err != nil {
+		return err
+	}
+
+	dirPath := filepath.Join(path, ReplicaDirectory)
+	filePath := filepath.Join(path, DiskConfigFile)
+
+	// Check if the replica directory exist, delete it
```

Comment:
Don't use `rm -rf`. It's not safe. Use `rmdir` instead. If it's created by us, it should be empty.